app/javascript/src/components/Dashboard/Contacts/Form.jsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+
+import { Input, Select } from "neetoui/v2/formik";
+
+import { ROLE_OPTIONS } from "./constants";
+
+const Form = () => {
+  const roleOptions = ROLE_OPTIONS.map(({ name, id }) => ({
+    label: name,
+    value: id,
+  }));
+
+  return (
+    <div className="w-full px-2 space-y-6">
+      <div className="flex space-x-5">
+        <Input
+          required
```

Comment:
I am assuming that you are you going to embed this whole `Form` component under `Formik`. Thus couldn't yup validation itself make this field required? Why add `required` explicitly over here?